pallets/asset-manager/src/lib.rs

```diff
@@ -296,15 +306,11 @@ pub mod pallet {
     pub(super) type LocationAssetId<T: Config> =
         StorageMap<_, Blake2_128Concat, T::Location, T::AssetId>;
 
-    /// AssetId to AssetRegistry Map.
+    /// AssetId to AssetMetadata Map.
     #[pallet::storage]
     #[pallet::getter(fn asset_id_metadata)]
-    pub(super) type AssetIdMetadata<T: Config> = StorageMap<
-        _,
-        Blake2_128Concat,
-        T::AssetId,
-        <T::AssetConfig as AssetConfig<T>>::AssetRegistryMetadata,
-    >;
+    pub(super) type AssetIdMetadata<T: Config> =
```

Comment:
does this even need to be (super) what is accessing it ?

Reply:
Just a way to explicitly disallow outside modules from accessing storage. `pub(super)` is more restrictive than just `pub`, I think it is good idea for a piece of storage like this

Reply:
I mean does it even need to be pub ?

Reply:
Well then it is out of scope anywhere outside `pub mod pallet`. Most of the time it is needed to be in scope for a test or maybe some  trait implementation

(This particular example is now reverted)